### ISSUE TITLE
eGPU support of AKiTiO Node Titan by purge-wrangler

### ISSUE THREAD

**Issue by madmacscientist** [mayankk2308/purge-wrangler#67]
### Feature Description
I have run the latest purge-wrangler on the 2013 Mac Pro. All the eGPUs work except the new Node Titan from OWC (MacSales.com) 

### Potential Solution
Not sure why it is ignored. Sometimes connecting and power up causes kernel panics.

### Alternatives
n/a

### More Context
Running macOS Catalina 10.15.4.

**Comment by mayankk2308** [comment]
Make sure to connect eGPU to bottom two ports only for 2013 Mac Pro. If using Navi GPU, set up system without connecting eGPU.

**Comment by madmacscientist** [comment]
Thank you for the advice.

I have been using the bottom two TB2 ports of the 2013 Mac Pro.

I’ve successfully used eGPU boxes from Sonnet, AKiTIO, and OWC on the 2013 Mac Pro with the help of purge-wrangler.

This latest box, the AKiTiO Node Titan, is new. It uses the AKiTiO Node Pro box with a different logic board and beefier PSU. I have had Navi GPUs (Pro W5700 and RX 5700 XT) running in the AKiTiO Node Pro and OWC Mercury Helios FX 650. 

But when I move the Navi GPU or VII to the AKiTiO Node Titan, it does not mount. OWC engineering is concerned and asked for a copy of my procedure since they are trying to troubleshot the issue. I sent them my steps along with a link to purge-wrangler page.

This morning I also tried purge-wrangler’s System Diagnosis & Logging. 

I forgot to attach the generated log when I posted the issue. I’ve attached two log files to this email.

I’ve decided to run Uninstall and try again.

Rob-ART 
BareFeats.com

**Comment by madmacscientist** [comment]
I just finished running purge-wrangler uninstall.

Restart.

Ran install.

When prompted to Plug-in eGPU, connected AKiTiO Node Titan eGPU to lower left TB2 port of 2013 Mac Pro.

Script responded with Detection failed.

I continued with the rest of the install operation including selecting AMD.

Restart

The Node Titan eGPU did not mount.

Switched to the Node Pro. It mounted.

Rob
BareFeats.com <http://barefeats.com/>

**Comment by mayankk2308** [comment]
I don't see the logs. Please post them on Github directly. It is possible that Node Titan is not being detected by macOS. Connect Node Titan and check in System Information > Thunderbolt. It should be present there. If it is, then GPU inside is not being detected.

**Comment by madmacscientist** [comment]
When I moved the TB2 cable with TB3 adapter from the Node Pro into to the Node Titan and powered it up, it does not show in System Information > Thunderbolt. I opened the top of the Node Titan. The RX 5700 XT GPU fan was not turning.

I powered off the Node Titan.

I unplugged the TB2 cable from the 2013 Mac Pro’s TB2 port.

I restarted the 2013 Mac Pro.

I powered up the Node Titan. Plugged the TB2 cable back into the lower left TB2 port on the 2013 Mac Pro.

The RX 5700 XT GPU fan started spinning.
 
I checked System Information > Thunderbolt. It now sees the Node Titan and the GPU is mounted.

Normally that sequence is not required but the important thing is that it now works.

Thanks for your support and suggestions.

Rob-ART
BareFeats.com
